lib/evse_security/crypto/openssl/openssl_supplier.cpp

```diff
@@ -85,36 +83,9 @@ const char* OpenSSLSupplier::get_supplier_name() {
     return OPENSSL_VERSION_TEXT;
 }
 
-bool OpenSSLSupplier::supports_tpm() {
```

Comment:
The function should be present, must conform to the interface.

Reply:
it is in the hpp file.